src/frontend/src/flows/verifiableCredentials/allowCredentials.ts

```diff
@@ -1,8 +1,14 @@
 import { mkAnchorInput } from "$src/components/anchorInput";
 import { mainWindow } from "$src/components/mainWindow";
 import { I18n } from "$src/i18n";
-import { mount, renderPage } from "$src/utils/lit-html";
-import { TemplateResult, html } from "lit-html";
+import { mount, renderPage, TemplateElement } from "$src/utils/lit-html";
+import { Chan } from "$src/utils/utils";
+import { html, TemplateResult } from "lit-html";
+import { asyncReplace } from "lit-html/directives/async-replace.js";
+import { unsafeHTML } from "lit-html/directives/unsafe-html.js";
+
+import DOMPurify from "dompurify";
+import { parse as parseMarked } from "marked";
```

Comment:
@peterpeterparker want me to lazy load this? in practice, in this flow this will always be imported

Reply:
Thanks for thinking at it. Agree it's fine given the use case and the fact that we ship on the IC ("often faster a big chunk of js than chunks").